modules/apps/commerce/commerce-product-asset-categories-web/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"dependencies": {
+		"frontend-js-web": "*"
+	},
+	"name": "@liferay/commerce-product-asset-categories-web",
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix",
+		"test": "liferay-npm-scripts test"
```

Comment:
@kresimir-coko this line is causing the CI failure, it is trying to run frontend tests and we don't have any. We can just remove it.

Reply:
👍  thank you!